pihole-google.txt

```diff
@@ -110,6 +110,7 @@ youtube.googleapis.com
 zfgbxkotjxujhmpo.storage.googleapis.com
 # Android
 1-market.android.com
+2.android.pool.ntp.org
```

Comment:
```suggestion
```
ntp.org is not a Google domain, see https://whois.domaintools.com/ntp.org

Reply:
Done